.github/workflows/docker.yml

```diff
@@ -0,0 +1,55 @@
+name: Build & Push
```

Comment:
Maybe "Docker Build & Push"?

Reply:
I didn't put docker here because the naming in ci would be `Docker/Docker Build & Push`, but can add it in if you think it's better. 

This is annoying with github actions. In ci it shows `<file_name>/<action_name>` but in the actions tab it only shows `action_name`.

Reply:
Oh, right, ok - yeah, that is annoying.